app/Http/Controllers/User/UserController.php

```diff
@@ -85,12 +86,25 @@ public function update(Request $request)
     /**
      * Removing a user from the system.
      *
-     * @param  int  $id
-     * @return \Illuminate\Http\Response
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse|\Symfony\Component\HttpFoundation\Response
      */
-//    public function destroy($id)
-//    {
-//
-//    }
+    public function destroy(Request $request)
+    {
+        $user = $request->user();
+
+        if ($user->transfersOut()->where('status', 'pending')->count()) {
+            return $this->buildFailedValidationResponse($request, 'You must complete all transfers.');
+        }
+
```

Comment:
No this does not include pending transfers going inward. There is not record of transfers going in and is only stored on the client side.

> Anyone who has never made a mistake has never tried anything new. -Albert Einstein